common/djangoapps/student/models.py

```diff
@@ -2744,3 +2744,24 @@ class Meta(object):
         db_table = "auth_accountrecovery"
 
     objects = AccountRecoveryManager()
+
+
+class CourseEnrollmentBanned(models.Model):
+    """
+    This table contains user, course_Id, Time_till_banned
+
+    Purpose of this table is to provide information of users
+    that are banned from a course for certain/noncertain amount of time.
+    """
+    email = models.CharField(max_length=255, db_index=True)
+    course_id = CourseKeyField(db_index=True, max_length=255, blank=True)
+    is_banned = models.BooleanField(default=1)
+    expiry = models.DateTimeField(null=True, blank=True, db_index=True)
+    created = models.DateTimeField(auto_now_add=True, null=True, db_index=True)
+    updated = models.DateTimeField(auto_now=True, db_index=True)
```

Comment:
why are we indexing these fields? we are not even searching on these fields, indexing every column will be costly in term of performance